test_rclcpp/test/test_n_nodes.py.in

```diff
@@ -52,13 +52,4 @@ class TestNNodes(unittest.TestCase):
 class TestNNodesAfterShutdown(unittest.TestCase):
 
     def test_@TEST_NUM_NODES@_nodes(self, proc_info):
-        launch_testing.asserts.assertExitCodes(
-            proc_info,
-            [launch_testing.asserts.EXIT_OK],
-            'node_check_names'
-        )
-        for i in range(@TEST_NUM_NODES@):
-            launch_testing.asserts.assertExitCodes(
-                proc_info,
-                process='node_with_name_' + str(i),
-            )
+        launch_testing.asserts.assertExitCodes(proc_info)
```

Comment:
Instead of asserting exit codes for each process individually, the new code calls assertExitCodes once for proc_info, which likely simplifies the test by assuming all necessary checks are included in proc_info.

Reply:
Yup. I rewrote this for some reasons.

1. Track the exit code by process objects rather than process name. 
2. If we follow the previous pattern, it's just have two parts of the check: one for the checking node process and one for the node_with_name processes, which I believe it's no harm to merge them into once.